src/lib/ui/LoadingToast/LoadingToast.tsx

```diff
@@ -0,0 +1,58 @@
+import classNames from 'classnames'
+import { useEffect, useState } from 'react'
+
+import { CheckBlackIcon } from 'lib/icons/CheckBlack.js'
+import { Spinner } from 'lib/ui/Spinner/Spinner.js'
+
+interface LoadingToastProps {
+  isLoading: boolean
+  isDoneLoading?: boolean
+  top?: number
+  left?: number
+}
+
+export function LoadingToast({
+  isLoading,
+  isDoneLoading = false,
```

Comment:
Why do we need two props? Can we just have `isLoading` and handle the transition via timeout with one useEffect?